### PR TITLE
A couple of small changes around the new query server

### PR DESCRIPTION
**Pull request by aeisenberg**
@alexet These are minor cleanups to your PR #1623.

### CODE REVIEW

File: extensions/ql-vscode/src/run-queries-shared.ts [hunk 1]
```diff
@@ -79,9 +79,6 @@ export class QueryEvaluationInfo {
     return path.join(this.querySaveDir, 'results.dil');
   }
 
-  /**
-   * Get the path that the compiled query is if it exists. Note that it only exists when using the legacy query server.
```
Comment (alexet) [inline extensions/ql-vscode/src/run-queries-shared.ts:83]: Why did you remove this comment?

**Review by alexet — APPROVED** [review]
Github seems confused. I miised that this was against my PR originally sorry. Other than the comment which I think is still true this LGTM.

File: extensions/ql-vscode/src/run-queries-shared.ts (hunk 1 above)
Comment (aeisenberg) [reply extensions/ql-vscode/src/run-queries-shared.ts:83]: I can add it back. I had originally thought that the new query server would be working similarly to the old one and create a qlo file, but I understand now that it will create the dil file directly.

File: extensions/ql-vscode/src/run-queries-shared.ts [hunk 2]
```diff
@@ -79,9 +79,6 @@ export class QueryEvaluationInfo {
...
-   */
   get compileQueryPath() {
```
Comment (aeisenberg) [inline extensions/ql-vscode/src/run-queries-shared.ts:82]:
```suggestion
  /**
   * Get the path that the compiled query is if it exists. Note that it only exists when using the legacy query server.
   */
    get compileQueryPath() {
```
